### ISSUE TITLE
Different methodology to resource change rate

### ISSUE THREAD

**Issue by neilkakkar** [Zac-Garby/solar-system#58]
I think instead of a random number in a fixed range being added each time, like in the case of Wood, Water etc, and instead of having current food exactly inversely related to square root of population, we ought to have a rate of production of each of these on the planet. Along with a max cap for each resource

Why this helps -
1. The growth of each resource is definite.
2. Say when we have scientists researching technologies to increase the production on planets, all we have to do is increase this rate to show higher production.
3. Say when we have upgraded storage on a planet, we just need to increase the max cap of the resource
4. Say someone notices that rate of growth of food is less than rate of growth of population, then they can make the deduction that the food would soon run out and take necessary measures instead of depending on a random increase in either.
5. In view of above points, helps scale when we add functionalities.

**Comment by neilkakkar** [comment]
Will work on this if the issue resonates with the team

**Comment by thecouchcoder** [comment]
@neilkakkar  I think this is related to issue #45 I was working on with the people management pane.  The idea was based off population to be able to assign a certain number of people to different jobs to increase/decrease production based off your liking.  You'll find a todo in the code of planet.cpp.  I had begun to try to setup a people management pane, but if you look at that open issue, I never got it worked out.


> // TODO: would like to implement people management ex. # of framers, scientists, engineers, 
> // laborers and resource production would be based of this

**Comment by zac-garby** [comment]
@neilkakkar Good idea. It'd certainly make it easier to decide what to do in terms of sending ships to other planets.

@aes421 About your issue: I found a fix a while back, but I'm just seeing now that I forgot to mention you in my comment
